databallpy/load_data/tracking_data/tracab.py

```diff
@@ -0,0 +1,264 @@
+from typing import Tuple
+
+import bs4
+import numpy as np
+import pandas as pd
+from bs4 import BeautifulSoup
+from tqdm import tqdm
+
+from databallpy.load_data.metadata import Metadata
+
+
+def load_tracking_data_tracab(
+    tracab_loc: str, meta_data_loc: str, verbose: bool = True
+) -> Tuple[pd.DataFrame, Metadata]:
+    """Function to load tracking data and metadata in the tracab format
+
+    Args:
+        tracab_loc (str): location of the tracking_data.dat file
+        meta_data-loc (str): location of the meta_data.xml file
+        verbose (bool): whether to print info, defaults to True
```

Comment:
Maybe a bit more elaborate, info about the progress of loading the file in the terminal